backend/src/core/middlewares/auth.middleware.ts

```diff
@@ -101,9 +132,19 @@ const isCookieOrApiKeyAuthenticated = async (
       // To avoid these checks, we assign the user id to the session property instead so that downstream middlewares can use it
       req.session.user = user
       req.session.apiKey = true
+      logger.info({
+        message: 'User authenticated by API key',
+        email: user.email,
+        id: user.id,
+        action: 'isCookieOrApiKeyAuthenticated',
+      })
       return next()
     }
 
+    logger.error({
```

Comment:
This doesn't provide much info... we don't know who provided the invalid api key / didn't provide an api key / was not a user / did not have a ssession. I don't think this needs to be logged actually